test/clique.js

```diff
@@ -12,22 +12,21 @@
 
 const eth_util = require('ethereumjs-util');
 const utils = require('./helpers/utils.js');
+const benchmark= require("solidity-benchmark")
 const encoder = require('./helpers/encoder.js');
 const Web3 = require('web3');
 const Web3Utils = require('web3-utils');
 const rlp = require('rlp');
-const truffleAssert = require('truffle-assertions');
 const sha3 = require('js-sha3').keccak_256
-
 const Clique = artifacts.require("Clique");
 const MockIon = artifacts.require("MockIon");
 const MockStorage = artifacts.require("MockStorage");
 
 const web3 = new Web3();
 const rinkeby = new Web3();
 
-web3.setProvider(new web3.providers.HttpProvider('http://localhost:8545'));
-rinkeby.setProvider(new web3.providers.HttpProvider('https://rinkeby.infura.io'));
+web3.setProvider(new web3.providers.HttpProvider('http//localhost:8545'));
```

Comment:
Do we use this provider? I thought we did use this. This should be failing due to missing colon `:`.

Reply:
You are right - but we are not using it so it doesn't fail. I'll delete it